core/templates/dev/head/components/forms/AudioTranslationsEditorDirective.js

```diff
@@ -75,6 +75,18 @@ oppia.directive('audioTranslationsEditor', [
               .isFullyTranslated($scope.contentId);
           };
 
+          $scope.getNeedsUpdateTooltipMessage = function(needsUpdate) {
+            if (needsUpdate) {
+              return ($scope.isEditable() ? 'Audio might not match text.' +
+                ' Reupload the file, or click to unflag.' :
+                'Audio might not match text.');
+            } else {
+              return ($scope.isEditable() ? 'Click to mark this audio ' +
+                'translation as not matching text.' :
+                'Audio is matching text.');
```

Comment:
Probably better to say "Audio translation matches text."

Reply:
Done!